### PR TITLE
update(config): replaced autobumper script with Kubernetes generic-autobumper.

### LINKED ISSUE #627: Switch to kubernetes image-bumper instead of our autobumper script
**Motivation**

Our autobumper script is written in bash.

**Feature**

 https://github.com/kubernetes/test-infra/tree/master/prow/cmd/generic-autobumper is written in go and it is surely more maintenable.

**Alternatives**

**Additional context**

### PR DESCRIPTION
**Pull request by cappellinsamuele**
Fixes #627.
Once this PR gets merged the [`autobump`](https://github.com/falcosecurity/test-infra/tree/master/images/autobump) image should be removed.

### CODE REVIEW

File: config/jobs/autobump/autobump.yaml [hunk 1]
```diff
@@ -1,73 +1,56 @@
...
+      - name: ssh
+        secret:
+          secretName: poiana-ssh-keys
+          defaultMode: 0400
+      nodeSelector:
+        Archtype: "x86"
+    annotations:
+      description: runs autobumper to create/update a PR that bumps prow to the latest RC without label 'skip-review'
+
+postsubmits:
```
Comment (FedeDP) [inline config/jobs/autobump/autobump.yaml:35]: I don't think we need to redeploy prow through this (we already have a circleci job, soon to be gha workflow, for that).
@maxgio92

Comment (cappellinsamuele) [reply config/jobs/autobump/autobump.yaml]: Fixed ;)

File: config/jobs/autobump/autobump.yaml [hunk 2]
```diff
@@ -1,73 +1,34 @@
...
+  - cron: "05 15 * * 1"  # Run at 15:05 PST (15:05 UTC) Mon
+    name: ci-test-infra-autobump-prow
+    decorate: true
+    extra_refs:
+    - org: falcosecurity
+      repo: test-infra
+      base_ref: master
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/generic-autobumper::v20230509-5e039b951c
```
Comment (maxgio92) [inline config/jobs/autobump/autobump.yaml:11]: I think there is a typo in the container image tag reference (`::`). Am I missing someting @cappellinsamuele?

**Review by maxgio92 — APPROVED** [review]
/approve

**Pull request merged**